scripts/start-ranger-admin.sh

```diff
@@ -0,0 +1,92 @@
+#!/bin/bash
+set -e
+source /ranger-common.sh
+
+cd ${RANGER_ADMIN_HOME}
+[[ -z $rangerAdmin_password ]] && export rangerAdmin_password=$(vault read -field=rangerAdmin_password ${vault_path}/ranger_admin)
```

Comment:
Can we use all uppercase for Unix env variable as used at other places?

Reply:
same as above, these are ranger defined variables.